soh/src/overlays/actors/ovl_En_Niw_Lady/z_en_niw_lady.c

```diff
@@ -455,7 +455,7 @@ void func_80ABAC00(EnNiwLady* this, GlobalContext* globalCtx) {
             getItemId = !(gSaveContext.itemGetInf[2] & 0x1000) ? GI_POCKET_EGG : GI_COJIRO;
 
             if (gSaveContext.n64ddFlag && getItemId == GI_POCKET_EGG) {
-                getItemId = Randomizer_GetItemIdFromKnownCheck(RC_KAK_ANJU_AS_ADULT, GI_POCKET_EGG);
+                getItemId = Randomizer_GetItemFromKnownCheck(RC_KAK_ANJU_AS_ADULT, GI_POCKET_EGG).getItemId;
```

Comment:
Need a comment here telling us to come back to this section for rando-next merge, as we'll need to make similar changes to the Cojiro check as well.